### PR TITLE
Convert LOnsager

### PR DESCRIPTION
**Pull request by etpalmer63**
Sorry this PR got out of hand. I believe it contains several of the others.

I'm currently working on debugging a difference in `ComputeDBarLocal` the non-default cases. For some reason the values come out a little different (seems like more than just system differences). 

To me the rest of the code changes look good at this point.

### CODE REVIEW

**Comment by etpalmer63** [comment]
Ready for review :grin:  

As I started to delete and remove, I discovered there is another routine to do.

**Comment by ajnonaka** [comment]
Which routine?  Is that the last one?

File: exec/multispec/inputs_rtil_spinodal_charges_2d [hunk 1]
```diff
@@ -18,8 +18,8 @@
   fixed_dt = 2.e-13
 
   ! Controls for number of steps between actions
-  max_step = 50000
-  plot_int= 1000
+  max_step = 20
+  plot_int= 10
```
Comment (ajnonaka) [inline exec/multispec/inputs_rtil_spinodal_charges_2d:22]:
```suggestion
  max_step = 50000
  plot_int= 1000
```

**Comment by ajnonaka** [comment]
This passed all 7 of the regression tests.  I think it's good to go.

**Pull request merged**